migrator/init.lua

```diff
@@ -35,77 +35,80 @@ end
 
 local function init()
     local httpd = cartridge.service_get('httpd')
-    httpd:route({ path = '/migrations/up', method = 'POST' }, function(req)
-        local target_names = get_diff()
-
-        if #target_names == 0 then
-            log.info('No migrations to apply!')
-            local resp = req:render({ json = { applied = {} }})
-            resp.status = 200
-            return resp
-        end
+    if httpd then
```

Comment:
`if not httpd then return true end` would be more readable, imo

Reply:
Fixed